portfolio/src/main/webapp/script.js

```diff
@@ -1,28 +1,41 @@
-// Copyright 2019 Google LLC
-//
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//
-//     https://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
+//A simple script to update the content of an html element with facts about me
+//depending on which element is clicked by a user
+//Author: Andrew Wiedenmann
+"use strict";
 
-/**
- * Adds a random greeting to the page.
- */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+//A map of displayKey values mapped to html string content. When an element is clicked, the string corresponding to that
+//displayKey value is displayed on the webpage
+let popupContentMap = new Map();
 
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
+let PHILOSOPHY_STR_CONTENT = "Right now, I'm reading  'On the Aesthetic Education of Man' by Friedrich Schiller. It's where I found the quote at the top of the page.";
```

Comment:
I think these are preferred to be const.

https://google.github.io/styleguide/jsguide.html#features-use-const-and-let

Reply:
done